nilearn/glm/io.py

```diff
@@ -334,7 +341,7 @@ def save_glm_to_bids(
             height_control=report_kwargs["height_control"],
         )
         table_details = clustering_params_to_dataframe(
-            report_kwargs["threshold"],
+            threshold,
```

Comment:
this should actually fix a bug that did not update the threshold before saving the cluster table to disk